qemu/tests/cfg/rng_bat.cfg

```diff
@@ -7,19 +7,11 @@
     Windows:
         # Please update path of rng_dll_register_cmd to right path which included you driver
         #rng_dll_register_cmd = if not exist "C:\Windows\system32\viorngum.dll" copy PATH:\INCLUDEDRIVER\viorngum.dll C:\Windows\system32\ /y &&"
-        #rng_dll_register_cmd += "rundll32 C:\Windows\system32\viorngum.dll,RegisterProvider"
         session_cmd_timeout = 240
         read_rng_cmd  = "X:\random_%PROCESSOR_ARCHITECTURE%.exe"
-        enable_verifier_cmd = "verifier.exe /standard /driver viorng.sys"
-        driver_verifier_cmd = "verifier.exe /querysettings"
-        driver_name = "viorng.sys"
+        driver_name = "viorng"
         rng_data_rex = "0x\w"
-        need_reboot = yes
     Linux:
         session_cmd_timeout = 360
         read_rng_cmd  = "dd if=/dev/hwrng  bs=1 count=10 2>/dev/null|hexdump"
```

Comment:
use /dev/hwrng , the command will hang there, replace it with /dev/random ,then works fine.